src/esm.ts

```diff
@@ -94,6 +94,17 @@ export function genExport(
   return _genStatement("export", specifier, exports, options);
 }
 
+/**
+ * Generate an ESM `export default` statement.
+ *
+ * @group ESM
+ */
+export function genExportDefault(
+  specifier: string,
+) {
+  return `export default ${specifier};`;
+}
```

Comment:
we already have:

```ts
genExport('pkg', ['default'])
// produces: export { default } from "pkg";
```

Reply:
`export default` but it doesn't give output this way